site/dat/docs/Reporting.md

```diff
@@ -202,6 +202,7 @@ modules:
                          # can be "start", "end", "both", "none"
     send-interval: 30s   # send data each n-th second
     timeout: 5  # connect and request timeout for BlazeMeter API
+    max-size: 10  # limit max size of file (in megabytes) that goes into zip for artifact upload, 100 by default
```

Comment:
Option name should be something more upload-specific. Like artifact-upload-size-limit.